devices/tuya.js

```diff
@@ -2975,7 +2975,7 @@ module.exports = [
         fromZigbee: [fz.ZG204ZL_lms],
         toZigbee: [tz.ZG204ZL_lms],
         exposes: [
-            e.occupancy(), e.illuminance(), e.battery(),
```

Comment:
Please use `e.illuminance().withUnit(`lx`)` instead. This device exposes a `illuminance` property, not `illuminance_lux`.